### DISCUSSION TITLE
How can I get this installed using Portainer?

### DISCUSSION THREAD

**Discussion by talormanda** [PlexRipper/PlexRipper#342] [Q&A]
I have all of my containers in Portainer Stacks. I don't see any docker compose documentation listed to help build that. Any advice?

**Comment by talormanda** [comment]
Edit: I see the instructions in the code. It should really be on a docs page or guides page.

**Reply by JasonLandbridge** [reply]
> It should really be on a docs page or guides page.

https://github.com/PlexRipper/Docs/issues/9 :thinking:

**Reply by talormanda** [reply]
Yeah sorry. It's not on Synology that I'm doing it this time. I have a new proxmox device with docker containers.

**Reply by JasonLandbridge** [reply]
My point is that you can't say "It should really be on a docs page or guides page." when you yourself are not putting in the effort of expanding the docs/guides when you promised it.

**Reply by talormanda** [reply]
I can add that when I get time. Synology changed their docker app so it's completely different now.

**Reply by talormanda** [reply]
Edit: It's 5AM, but I have it running in synology even though I don't use that anymore. I will work on getting a quick few words typed up later.
